tecan_od_analyzer/tecan_od_analyzer.py

```diff
@@ -874,8 +909,8 @@ def gr_estimation(df_gr_final):
         # Dataframe generation with outlier free data
         df_temp = pd.DataFrame(
             {
-                colnames[col] + "_data": est_series.index,
-                colnames[col] + "_time": est_series.values,
+                colnames[col] + "_time": est_series.index,
```

Comment:
Ooops :).  I will need to give our previous MSc intern a hard time about this one.